index.bs

```diff
@@ -3061,6 +3065,12 @@ or null and an [=struct/item=] named <dfn attribute for="viewport-configuration"
 
 A [=remote end=] has a <dfn>viewport overrides map</dfn> which is a weak map between [=user contexts=] and [=viewport configuration=].
 
+A [=remote end=] has a <dfn>navigable to locale overrides map</dfn> which is a weak
+map between [=navigables=] and strings.
+
+A [=remote end=] has a <dfn>user context to locale overrides map</dfn> which is a
+weak map between [=user contexts=] and strings.
```

Comment:
Lets make sure that the naming of the map is done similarly to other override maps (see above):

```suggestion
A [=remote end=] has a <dfn>locale overrides map</dfn> which is a
weak map between [=user contexts=] and strings.
```

Reply:
There are 2 maps: for user contexts and for navigable, and it is required to distinguish between them